modules/geolocation/unittests.py

```diff
@@ -194,8 +194,68 @@ def test_point_set_2(self):
         actual = locator.calculate_pixel_to_geo_mapping()
         # Test
         np.testing.assert_almost_equal(actual, expected)
+    
+class TestMapLocationFromPixel(unittest.TestCase):
+    """
+    Tests Geolocation.map_location_from_pixel()
+    """
+    def setUp(self):
+        
+        self.locator = geolocation.Geolocation()
+
+        return
+    
+    def test_identity_transformation_matrix(self):
+
+        # Setup
+        identityTransformationMatrix = np.ones(shape=(3,3))
+        pixelCoordinates = np.array([[2,3],[12,99],[623,126],[1604,12],[0,4]])
 
+        expected = np.array(np.ones(shape=(5,2)))
+
+        # Run
+        actual = self.locator.map_location_from_pixel(identityTransformationMatrix, pixelCoordinates)
         
+        # Test
+        np.testing.assert_almost_equal(actual, expected)
+    
+    def test_transformation_matrix_1(self):
+
+        # Setup
+        transformationMatrix = np.array([[4,6,152],[120,5,99],[3,5,2]])
+        pixelCoordinates = np.array([[2,3],[12,99],[623,126],[1604,12],[0,4]])
+
+        expected = np.array([[7.739130435,15.39130435], 
```

Comment:
Formatting nitpick: Try to keep it aligned.

Reply:
Changed such that all rows of the matrices are on a new line:

`transformationMatrix = np.array([[5,1,5],
                                                       [9,1,2],
                                                       [0,0,0]])
`